systemtest/pom.xml

```diff
@@ -200,10 +200,44 @@
             <artifactId>bcprov-jdk15on</artifactId>
             <version>${bouncycastle.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.sundr</groupId>
+            <artifactId>builder-annotations</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
In the api module this is the only sundr dependency required. We don't need to explicitly declare a dep on `sundr-core` or `sundr-codegen`